frontend/src/scenes/notebooks/Nodes/NotebookNodePlaylist.tsx

```diff
@@ -63,6 +68,7 @@ const Component = (props: NodeViewProps): JSX.Element => {
             title="Session Replays"
             href={urls.replay(undefined, filters)}
             heightEstimate={HEIGHT}
+            compact={isShared}
```

Comment:
If node shareability was decided in the wrapper we would not need to pass this compact flag in